### DISCUSSION TITLE
Enable color compilation

### DISCUSSION THREAD

**Discussion by kammce** [friendlyanon/cmake-init#27] [Q&A] [answered]
When I run `cmake --build build/dev` my output form the compiler does not have any color. Whats the preferred way to bring the color back.

**Comment by friendlyanon** [comment]
`cmake --build ...` simply invokes the underyling build system, so it depends on that. The example GIF in the README shows me invoking a Unix Makefiles based build with color output.

**Comment by kammce** [comment, marked as answer]
I ended up just adding `UseColor: true` to the `.clang-tidy` file which added color to that output. As for the compiler output, it doesn't seem to come out because clang-tidy detects errors first.
